lib/session-file-store.js

```diff
@@ -0,0 +1,74 @@
+const fse = require('fs-extra')
+const path = require('path')
+const Store = require('express-session').Store
+
+class FileStore extends Store {
+  constructor (options) {
+    super()
```

Comment:
Do we need to pass `options` into `super()`?

Reply:
No we don't, not for our file store